sklearn/metrics/tests/test_ranking.py

```diff
@@ -975,3 +975,148 @@ def test_ranking_loss_ties_handling():
     assert_almost_equal(label_ranking_loss([[1, 0, 0]], [[0.25, 0.5, 0.5]]), 1)
     assert_almost_equal(label_ranking_loss([[1, 0, 1]], [[0.25, 0.5, 0.5]]), 1)
     assert_almost_equal(label_ranking_loss([[1, 1, 0]], [[0.25, 0.5, 0.5]]), 1)
+
```

Comment:
Could you also compare your implementation with a naive implementation:
for each sample, sort the scores and count the number correct labels in the top-k.

This will allow to test correctness on less trivial examples.